I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 13, in script
    $ renpy.show(Envi/room_mc.png)
  File "game/script.rpy", line 13, in <module>
    $ renpy.show(Envi/room_mc.png)
NameError: name 'Envi' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 13, in script
    $ renpy.show(Envi/room_mc.png)
  File "D:\RenPy\renpy-8.3.0-sdk\renpy\ast.py", line 821, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\RenPy\renpy-8.3.0-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 13, in <module>
    $ renpy.show(Envi/room_mc.png)
NameError: name 'Envi' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.3.0.24082114
Capstone 1.0
Fri Nov 15 20:11:21 2024
